src/view/dataElements/xdmObject.jsx

```diff
@@ -115,22 +142,32 @@ const XdmObject = ({
               <FieldLabel labelFor="schemaField" label="Schema" />
             </InfoTipLayout>
           </div>
-          <div>
-            <Select
+          <div className="u-flex">
+            <ComboBox
               id="schemaField"
               data-test-id="schemaField"
               className="u-widthAuto u-gapBottom u-fieldLong"
               options={schemaOptions}
-              value={selectedSchemaMeta.$id}
-              onChange={schemaMetaId => {
+              value={schemasMetaSearch}
+              showMenu={showComboMenu}
+              onChange={search => {
+                setSchemasMetaSearch(search);
+                doSearch({ sandboxName: selectedSandbox.name, search });
+              }}
+              onSelect={item => {
                 setSelectedNodeId(undefined);
                 setSelectedSchemaMeta(
-                  schemasMeta.find(
-                    schemaMeta => schemaMeta.$id === schemaMetaId
-                  )
+                  schemasMeta.find(schemaMeta => schemaMeta.$id === item.value)
                 );
               }}
+              onMenuToggle={menuState => {
+                setShowComboMenu(menuState);
+              }}
+              placeholder="Search for or select a schema"
             />
+            <div className="u-gapTop u-gapLeft">
```

Comment:
Rather than using `u-gapTop` here, I would recommend using `u-alignItemsCenter` on the div that contains this div. It will probably result in more accurate centering and is more robust if the size of the Wait or ComboBox changes.